src/NUnitTestAdapterTests/TestExecutionTests.cs

```diff
@@ -153,9 +158,48 @@ public void TestResultIsReportedCorrectly(string name, TestOutcome outcome, stri
         }
 
         [Test]
-        public void ExplicitTestDoesNotShowUpInResults()
+        public void AttachmentsShowUpInResults()
         {
-            Assert.Null(testResults.Find(r => r.TestCase.DisplayName == "ExplicitlyRunTest"));
+            TestResult test = testResults.Find(r => r.TestCase.DisplayName == "SingleAttachmentTest");
```

Comment:
Minor: This and the below could perhaps use `nameof()` - in case of future refactoring. 🙂